data/src/main/java/org/oppia/android/data/backends/gae/api/PlatformParameterService.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.android.data.backends.gae.api
+
+import retrofit2.Call
+import retrofit2.http.GET
+import retrofit2.http.Query
+
+/** Service that provides access to the Oppia platform parameter endpoint. */
+interface PlatformParameterService {
+
+// TODO(#3506): Change the url to point to the correct endpoint when the backend is ready
+  @GET("platform_features_evaluation_handler")
+  fun getPlatformParametersByVersion(
+    @Query("app_version") version: String,
+    @Query("platform_type") platformType: String = "Android"
```

Comment:
We ought to verify that this is being set correctly via the mock.

Reply:
Done